cmake/FirstClassLangCuda.cmake

```diff
@@ -32,12 +32,12 @@ function(detect_cuDNN)
   set(CUDNN_ROOT "" CACHE PATH "CUDNN root folder")
 
   find_path(CUDNN_INCLUDE cudnn.h
-            PATHS ${CUDNN_ROOT} $ENV{CUDNN_ROOT}
+            PATHS ${CUDNN_ROOT} $ENV{CUDNN_ROOT} ${CUDA_TOOLKIT_ROOT_DIR}
             DOC "Path to cuDNN include directory." )
 
 
   find_library(CUDNN_LIBRARY NAMES libcudnn.so cudnn.lib # libcudnn_static.a
-                             PATHS ${CUDNN_ROOT} $ENV{CUDNN_ROOT} ${CUDNN_INCLUDE}
+                             PATHS ${CUDNN_ROOT} $ENV{CUDNN_ROOT} ${CUDNN_INCLUDE} ${CUDA_TOOLKIT_ROOT_DIR}
```

Comment:
what scenario is this needed to make it work ? I don't have CUDNN_ROOT set but it still is able to find CUDNN_LIBRARY. is it when cuda is installed to a custom dir outside /usr or /usr/local